store/config.go

```diff
@@ -0,0 +1,17 @@
+package store
+
+import (
+	"path"
+)
+
+type Config struct {
+	path string
+}
+
+const (
+	chatNamespace = "chat"
+)
+
+func NewConfig(dataDir string) *Config {
```

Comment:
This func should be called from `config/config.go` without any params. 
Everything inside Config is `settings` and should be loaded from config file e.g. `examples/config.yml`

Reply:
so what is the default configuration directory?

Reply:
https://github.com/pastelnetwork/go-commons/blob/c11ca9b4ae7547a8538710c798b158abb463e462/configurer/file.go#L11
It will be great if you examine the code of the https://github.com/pastelnetwork/go-commons/

Reply:
I don't need to set a default path, I need just GET

Reply:
Let me show you what I mean 
https://github.com/pastelnetwork/walletnode/compare/integration-code-example